src/aiotk/stack.py

```diff
@@ -20,25 +30,30 @@ class EnsureDone:
 
     """
 
-    def __init__(self, coro, cancel=True, loop=None):
+    # NOTE: the type of `coro` should be `typing.Coroutine` to reject futures,
+    #       tasks and other awaitables (`loop.create_task()` does not accept
+    #       them), but there is a bug in `mypy` which causes the return type of
+    #       coroutine functions to be reported as `typing.Awaitable`.
+    def __init__(self, coro: Awaitable, cancel=True, loop=None) -> None:
```

Comment:
**Note to self**: report this feedback to `mypy` devs to ensure they are aware of this limitation in `mypy`.